code/modules/mob/living/carbon/alien/special/alien_embryo.dm

```diff
@@ -84,26 +84,33 @@
 			return
 		attempt_grow()
 
-///Attempt to burst an alien outside of the host, getting a ghost to play as the xeno.
+/// Attempt to burst an alien outside of the host, getting a ghost to play as the xeno.
```

Comment:
For the record I can see changes to this file as possibly controversial. This prevents you from queueing from every alien embryo, but it also allows you to see your chances at spawning as one.